src/client/features/admin/index.js

```diff
@@ -174,4 +175,195 @@ class Admin extends React.Component {
   }
 }
 
+class Admin extends React.Component {
+  
+    constructor(props) {
+      super(props);
+  
+      const query = queryString.parse(props.location.search);
+  
+      this.state = {
+        query: _.assign({q: '', gt: 2, lt: 250, type: 'Pathway'}, query),
+        searchResults: [],
+        editLinks: [],
+        dataSources: []
+      };
+  
+      PathwayCommonsService.datasources()
+      .then(result => this.setState({
+        dataSources: result
+        })
+      );
+      this.updateEditLinks = this.updateEditLinks.bind(this);
+    }
+  
+    updateEditLinks(editURI) {
+      const editLinkObj = this.deconstructEditURI(editURI);
+      var editLinks = this.state.editLinks;
+      editLinks[editLinkObj.uri] = editLinkObj.editkey;
+      this.setState({
+        editLinks: editLinks
+      });
+    }
+  
+    deconstructEditURI(editURI) {
+      const id = '&editkey=';
+      const uri = editURI.substring(0, editURI.indexOf(id));
+      const editkey = editURI.substring(editURI.indexOf(id) + id.length, editURI.length);
+      return {uri: uri, editkey: editkey};
+    }
+
+    getSearchResult() {
+      const state = this.state;
+      const query = state.query;
+  
+      if (query.q !== '') {
+        PathwayCommonsService.querySearch(query)
+        .then(searchResults => {
+          this.setState({searchResults: searchResults}, () => {
+            CDC.initEditLinkSocket(this.updateEditLinks);
+            for (var i = 0; i < searchResults.length; i++) {
+              CDC.requestEditLink(searchResults[i].uri, 'latest');
+            }
+          });
+        });
+      }
+    }
+  
+    componentDidMount() {
+      this.getSearchResult();
+    }
+  
+    onSearchValueChange(e) {
+      // if the user presses enter, submit the query
+      if (e.which && e.which ===  13) {
+        this.submitSearchQuery(e);
+      } else {
+        const newQueryState = _.assign({}, this.state.query);
+        newQueryState.q = e.target.value;
+        this.setState({query: newQueryState});
+      }
+    }
+  
+    setQueryType(e, type) {
+      const newQueryState = _.assign({}, this.state.query);
+      newQueryState.type = type;
+      this.setState({query: newQueryState}, function () { this.submitSearchQuery(); });
+    }
+  
+    submitSearchQuery() {
+      const props = this.props;
+      const state = this.state;
+  
+      const query = state.query;
+      const uriRegex = /^(?:http(s)?:\/\/)?[\w.-]+(?:\.[\w\.-]+)+[\w\-\._~:/?#[\]@!\$&'\(\)\*\+,;=.]+$/;
+  
+      if (query.q.match(uriRegex)) {
```

Comment:
This was a legacy search feature that allowed someone to bypass the search and just input a pathway commons id such as "http://identifiers.org/reactome/R-HSA-6804186" and it will go straight to the view without returning search results

I just talked with Jeff about it and he says we can just remove this.